scripts/examples/openiotsdk_example.sh

```diff
@@ -113,6 +113,10 @@ function build_with_cmake() {
     PATH=$(echo "$PATH" | sed 's/:/\n/g' | grep -v "$PW_ARM_CIPD_INSTALL_DIR" | xargs | tr ' ' ':')
 
     cmake -G Ninja -S "$EXAMPLE_PATH" -B "$BUILD_PATH" --toolchain="$TOOLCHAIN_PATH" "${BUILD_OPTIONS[@]}"
+    if [ -f "$BUILD_PATH"/_deps/tf-m-src/tools/requirements.txt ]; then
```

Comment:
What does this do (i.e. why is this needed)?

If this shell script changes, should there be a change in `scripts/build/builders/openiotsdk.py` ?

Reply:
We need to install the required Python packages for TF-M tools which are used during its component build. I added some comments.
Yes, definitely, I added changes there.